app/presenters/charge_presenter.rb

```diff
@@ -1,9 +1,10 @@
 class ChargePresenter < BasePresenter
-  delegate :code,
```

Comment:
Removed this `code` delegation as we are not using it and is likely we will not need it after all.
It is of course possible to access it through the `offence` method, but from a presenter point of view it doesn't make sense to have it unless we are going to present it.

Reply:
That makes sense - confusing if we done use it for sure!